mobile/src/main/java/ch/epfl/sdp/mobile/application/tts/TextToSpeechFacade.kt

```diff
@@ -0,0 +1,7 @@
+package ch.epfl.sdp.mobile.application.tts
+
+import ch.epfl.sdp.mobile.infrastructure.tts.TextToSpeechFactory
+
+class TextToSpeechFacade(private val factory: TextToSpeechFactory) {
+    val synthesizer = suspend { factory.create() }
```

Comment:
The `synthesizer` should not be exposed directly through the facade. Instead, consider having a method `speech(text: String)` on the facade, which calls the `TextToSpeech` service.